test/ufuzz.js

```diff
@@ -135,23 +182,25 @@ var FUNC_TOSTRING = [
     "}();",
     ""
 ].join("\n");
+var loops = 0;
+var funcs = 0;
 
 function run_code(code) {
     var stdout = "";
     var original_write = process.stdout.write;
-    process.stdout.write = function(chunk) {
+    process.stdout.write = function (chunk) {
```

Comment:
(FWIW I've updated my editor for this project to use 4 instead of 2 tabs, which is the biggest reason for the fluctuating indents, so those shouldn't appear anymore. If there's other style guide rules to adhere I'll configure them for sure. I can also replace this commit with those style changes instead. Or just drop the commit if you prefer it that way. It's just an automatic beautification really.)